modules/nf-core/sentieon/bwaindex/main.nf

```diff
@@ -0,0 +1,69 @@
+process SENTIEON_BWAINDEX {
+    tag "$fasta"
+    label 'process_high'
+    label 'sentieon'
+
+    secret 'SENTIEON_LICENSE_BASE64'
+
+    // Exit if running this module with -profile conda / -profile mamba
+    if (workflow.profile.tokenize(',').intersect(['conda', 'mamba']).size() >= 1) {
+        exit 1, "Sentieon modules does not support Conda. Please use Docker / Singularity / Podman instead."
+    }
+
+    container 'nfcore/sentieon:202112.06'
+
+    input:
+    tuple val(meta), path(fasta)
+
+    output:
+    tuple val(meta), path(bwa), emit: index
```

Comment:
```suggestion
    tuple val(meta), path('bwa'), emit: index
```
I realise the unquoted thing works, here and in bwa/index, but I don't understand why! Seems like it should be the string.

Reply:
I copied the whole logic from the bwa/index as you guessed.
So for me as long as we don't update bwa/index, I won't change that here